src/react/components/pages/train/trainPage.tsx

```diff
@@ -438,7 +438,7 @@ export default class TrainPage extends React.Component<ITrainPageProps, ITrainPa
             const result = await ServiceHelper.postWithAutoRetry(
                 baseURL,
                 payload,
-                {},
+                {timeout:5400000},
```

Comment:
suggestion: there's already a `minimumTimeoutInMs` at line: 468 for timeout, we should update the timeout there.